DATA/EQUIPMENT/engine_equip.ini

```diff
@@ -4764,161 +4844,405 @@ inside_sound_cone = 60
 outside_sound_cone = 180
 lootable = true
 
-;; NOMADS
+;; Outcasts
 
 [Engine]
-nickname = no_engine_fighter_light01
-ids_name = 459042
+nickname = ou_engine_transport01
+ids_name = 459039
 ;res str
-; Nomad Propulsion Component
-ids_info = 460980
+; Outcast Transport Engine
+ids_info = 460977
 ;res html
-; \m\bNomad Propulsion Component\B
+; \m\bOutcast Engine\B
 ; \l
 ; Type: $classLong
-; Speed: -$reverseSpeed to $speed m/s
+; Speed: -16 to 80 m/s
 ; Cruise Speed: $cruiseSpeed m/s
 ; Cruise Charge Time: $cruiseChargeTime s
 ; Power Draw: $powerUsage /s
 ; Cruise Power Draw: $cruisePowerUsage /s
 ; Mass: $mass
 ; \
-; Bio-energy signatures detected by scanners. This engine system matches none in the database. The design is of alien origin.
+; eveloped by the recently founded Benitez Foundations, this engine drives every Outcast ship in Sirius.

```

Comment:
Eveloped. Missing the D at the beginning. Also, this does not drive every outcast ship in Sirius I am sure., It's a transport engine.